samples/graph-rsc/nodeJs/views/configure.ejs

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>RSC demo node js</title>
+    <style>
+        body {
+            margin: 0px;
+            padding: 0px;
```

Comment:
Could you please move styles to separate file?

Reply:
moved to other folder and using